diff --git a/Aurora/RuntimeCompiler/Compiler_PlatformWindows.cpp b/Aurora/RuntimeCompiler/Compiler_PlatformWindows.cpp
index eb138a78..bac5d159 100644
--- a/Aurora/RuntimeCompiler/Compiler_PlatformWindows.cpp
+++ b/Aurora/RuntimeCompiler/Compiler_PlatformWindows.cpp
@@ -61,7 +61,7 @@ struct CmdProcess
 	~CmdProcess();
 
 	void InitialiseProcess();
-	void WriteInput(std::string& input);
+	void WriteInput(std::string const& input);
 	void CleanupProcessAndPipes();
 
 
@@ -703,7 +703,7 @@ void CmdProcess::InitialiseProcess()
 	}
 	*/
 
-	const wchar_t* pCommandLine = L"cmd /q /K @PROMPT $";
+	static constexpr wchar_t const* pCommandLine = L"cmd /q /K @PROMPT $";
 	//CreateProcessW won't accept a const pointer, so copy to an array 
 	wchar_t pCmdLineNonConst[1024];
 	wcscpy_s(pCmdLineNonConst, pCommandLine);
@@ -740,7 +740,7 @@ ERROR_EXIT:
 }
 
 
-void CmdProcess::WriteInput( std::string& input )
+void CmdProcess::WriteInput( std::string const& input )
 {
 	DWORD nBytesWritten;
 	DWORD length = (DWORD)input.length();
 