diff --git a/code/AssetLib/glTF2/glTF2Exporter.cpp b/code/AssetLib/glTF2/glTF2Exporter.cpp
index 42b9c9fae..d1e8b7b51 100644
--- a/code/AssetLib/glTF2/glTF2Exporter.cpp
+++ b/code/AssetLib/glTF2/glTF2Exporter.cpp
@@ -683,7 +683,10 @@ bool glTF2Exporter::GetMatSheen(const aiMaterial &mat, glTF2::MaterialSheen &she
     }
 
     // Default Sheen color factor {0,0,0} disables Sheen, so do not export
-    if (sheen.sheenColorFactor == defaultSheenFactor) {
+        if (sheen.sheenColorFactor[0] == defaultSheenFactor[0]
+        && sheen.sheenColorFactor[1] == defaultSheenFactor[1]
+        && sheen.sheenColorFactor[2] == defaultSheenFactor[2]
+    ) {
         return false;
     }
 
